docs/index.rst

```diff
@@ -5,7 +5,7 @@ Monitoring Kafka streaming ETL deployments
 
 The Confluent Platform demo at `https://github.com/confluentinc/cp-demo <https://github.com/confluentinc/cp-demo>`__ shows users how to monitor Kafka streaming ETL deployments
 using `Confluent Control Center <http://docs.confluent.io/current/control-center/docs/index.html>`__.
-All the components in the |CP| have security enabled end-to-end. Follow along with the playbook in this README and watch the video tutorials.
+All the components in the |CP| have security enabled end-to-end. 
```

Comment:
Can we leave the line in but remove README?  For example: "Follow along with the playbook in this page and watch the video tutorials."

Reply:
in looking at this more, i think this first paragraph needs more work (e.g. it's not good information design to link people off to other pages in the first line). we want people reading about and using the demo from this page, not GitHub, so we should probably remove the link, at least in this prominent spot. i'll add to this PR.